src/classes/app.py

```diff
@@ -35,27 +35,28 @@
 
 from PyQt5.QtCore import PYQT_VERSION_STR
 from PyQt5.QtCore import QT_VERSION_STR
-from PyQt5.QtCore import Qt
+from PyQt5.QtCore import Qt, pyqtSlot
 from PyQt5.QtGui import QPalette, QColor, QFontDatabase, QFont
 from PyQt5.QtWidgets import QApplication, QStyleFactory, QMessageBox
 
 try:
-    # QtWebEngineWidgets must be loaded prior to creating a QApplication
-    # But on systems with only WebKit, this will fail (and we ignore the failure)
-    from PyQt5.QtWebEngineWidgets import QWebEngineView
-except ImportError:
+    # This apparently has to be done before loading QtQuick
+    # (via QtWebEgine) AND before creating the QApplication instance
+    QApplication.setAttribute(Qt.AA_ShareOpenGLContexts)
+    from OpenGL import GL  # noqa
+except (ImportError, AttributeError):
     pass
 
 try:
-    # Solution to solve QtWebEngineWidgets black screen caused by OpenGL not loaded
-    from OpenGL import GL
+    # QtWebEngineWidgets must be loaded prior to creating a QApplication
+    # But on systems with only WebKit, this will fail (and we ignore the failure)
+    from PyQt5.QtWebEngineWidgets import QWebEngineView
```

Comment:
I had originally moved these into `OpenShotApp.__init__()`, inside an `if mode != "unittest":` wrapper, because having them here was annoying the unit test runner. But, since these are things that need to be done **before** the QApplication instance is constructed, and we've already called `QApplication.__init__()` by that point, I don't think that counts as "before".

(The _real_ correct place to call these may be from inside `launch.py`, in fact. That would also solve things for the unit tests, since `query_tests.py` doesn't use `launch.py` so they'd never be called at all.)